jetstream/experimenter.py

```diff
@@ -205,7 +212,24 @@ def from_experimenter(cls, session: requests.Session = None) -> "ExperimentColle
                     str(e), exc_info=e, extra={"experiment": nimbus_experiment["slug"]}
                 )
 
-        return cls(nimbus_experiments + legacy_experiments)
+        draft_experiments = []
+        if with_draft_experiments:
+            # draft experiments are mainly used to compute previews
+            draft_experiments_json = retry_get(
+                session, cls.EXPERIMENTER_API_URL_DRAFTS, cls.MAX_RETRIES, cls.USER_AGENT
+            )
+
+            for draft_experiment in draft_experiments_json:
+                try:
+                    draft_experiments.append(
+                        ExperimentV6.from_dict(draft_experiment).to_experiment()
+                    )
+                except Exception as e:
+                    logger.exception(
+                        str(e), exc_info=e, extra={"experiment": nimbus_experiment["slug"]}
+                    )
```

Comment:
Is this going to end up in bigquery and the dashboards? If so, should we either a) do something different to avoid that (a different logger or just `print` for this?), or b) add a way to filter out draft experiments from the dashboards (another `extra` field?)?

Reply:
right. I think a simple print is sufficient here